epix/strax_interface.py

```diff
@@ -0,0 +1,92 @@
+import strax
+import time
+import os
+import epix
+import wfsim
+import numpy as np
+import pandas as pd
+import awkward1 as ak
+
+@strax.takes_config(
+strax.Option('input_file',
+             help='Input Geant4 ROOT file', track=True),
+strax.Option('entry_stop', type=int, default=None,
+             help='Number of entries to read from first. Defaulted to all'),
+strax.Option('events_per_chunk', type=int, default=200,
+             help='Number of events put in a chunk'),
+strax.Option('chunk_time_seperation', type=int, default=int(1e10),
+             help='Time seperation between chunks'),
```

Comment:
How does this work together with together with the event_rate which can be set? And how does this work together with WFSim, is WFSim reading in multiple chunks at a time? In the current implementation we would split interactions which are from the same event. Hence WFSim could not apply its chunking of instructions in the RawData-class, leading to possibly overlapping puleses etc.